src/content/development/contribution-guide/_index.en.md

```diff
@@ -46,7 +46,12 @@ To be expanded
 
 ## Release Cycles
 
-To be expanded
+The Submariner project follows time based release cycles, each cycle is 16 weeks long.
+Blocking bugs might cause a delay in the final release date, while new features will never be a reason to delay the release.
+
+Features that were partially implemented in a given release will be considered "experimental" and will not have any support commitment.
```

Comment:
```suggestion
A features that was partially implemented in a given release will be considered "experimental" and will not have any support commitment.
```

Reply:
I'd keep the original. I think it makes sense to be consistently plural here.